### PR TITLE
Run from cli

### PR DESCRIPTION
**Pull request by tsundvoll**

### CODE REVIEW

**Comment by tsundvoll** [comment]
This PR makes it so that instead of starting ISAR with 'python main.py', the application is started with the cli command 'isar'. I suggest the next releast to be a minor bump (1.20.0) to emphasis this change.

File: main.py [hunk 1]
```diff
@@ -1,99 +1,8 @@
-import logging
-import time
-from logging import Logger
-from threading import Thread
-from typing import List
+# This file is added for backwards-compability for versions 1.19.2 and older.
+# It can be removed in the next minor bump (1.20.0)
```
[the remark this answers is not in the document]
Comment (tsundvoll) [reply main.py:2]: Yes, that is the idea. i want to remove the Dockerfile and image altogether from ISAR and just build it from isar-robot (and the other sup-repos). Those already have isar available as the library and does not need the isar image

**Review by oysand — COMMENTED** [review]
LGTM

**Review by oysand — APPROVED** [review]
LGTM

**Pull request merged**